libraries/wallet/include/graphene/wallet/wallet.hpp

```diff
@@ -1310,11 +1310,15 @@ class wallet_api
        * @param owner_account the name or id of the account which is creating the SON
        * @param url a URL to include in the SON record in the blockchain.  Clients may
        *            display this when showing a list of SONs.  May be blank.
+       * @param deposit_id vesting balance id for SON deposit
+       * @param pay_vb_id vesting balance id for SON pay_vb
        * @param broadcast true to broadcast the transaction on the network
        * @returns the signed transaction registering a SON
        */
       signed_transaction create_son(string owner_account,
                                     string url,
+                                    vesting_balance_id_type deposit_id,
+                                    vesting_balance_id_type pay_vb_id,
```

Comment:
Please,  take a look here: are this argument types acceptable for wallet_api call?

Reply:
I think this is fine, users will use "1.13.x"as string as an argument to the call and it will be casted to `vesting_balance_id_type` all alone IIRC.